modules/javafx.graphics/src/main/java/com/sun/javafx/font/FallbackResource.java

```diff
@@ -332,4 +365,18 @@ public FontStrike getStrike(float size, BaseTransform transform,
         }
         return strike;
     }
+
+    public String toString() {
+        int ns = getNumSlots();
+        String s = "Fallback resource:\n";
+        for (int i=0;i<ns;i++) {
+          if ((getSlotResource(i)==null)) {
+           s+= "Slot " + i + "=null\n";
+          } else {
+          s += "Slot " + i + "="+getSlotResource(i).getFullName()+"\n";
+          }
+        }
+        s+= "\n";
```

Comment:
perhaps it might be better to use a StringBuilder here

Reply:
I don't think it matters - performance isn't an issue.